lib/main_common.pm

```diff
@@ -1594,6 +1594,7 @@ sub load_extra_tests_console {
     loadtest "console/lshw" if ((is_sle('15+') && (check_var('ARCH', 'ppc64le') || check_var('ARCH', 'x86_64'))) || is_opensuse);
     loadtest 'console/quota' unless is_jeos;
     loadtest 'console/zziplib' if (is_sle('12-SP3+') && !is_jeos);
+    loadtest 'console/firewalld';
```

Comment:
`firewalld` is available for SLE and Leap >=15. Suggested schedule will run test on all distributions and will fail on older SLE/Leap versions.

Reply:
Hi Petr, it's true. I'll correct the schedule and commit again. Thanks for spotting this.

Reply:
Scheduling for the test now includes a conditional to run only from SLE 15+, Tumbleweed or Leap